tests/unit/connect/render.cpp

```diff
@@ -16,8 +16,9 @@ using namespace json;
 namespace {
 
 constexpr const char *const print_file = "/usb/box.gco";
-constexpr const char *const rejected_event_printing = "{\"state\":\"PRINTING\",\"command_id\":11,\"event\":\"REJECTED\"}";
+constexpr const char *const rejected_event_printing = "{\"reason\":\"Job ID doesn't match\",\"state\":\"PRINTING\",\"command_id\":11,\"event\":\"REJECTED\"}";
```

Comment:
The last `const` is already implied by `constexpr`, to my knowledge `constexpr const char *` is exactly the same as `constexpr const char *const` except shorter